pilot/cmd/pilot-discovery/main.go

```diff
@@ -162,8 +162,8 @@ func init() {
 		"Discovery service HTTP address")
 	discoveryCmd.PersistentFlags().StringVar(&serverArgs.DiscoveryOptions.HTTPSAddr, "httpsAddr", ":15017",
 		"Injection and validation service HTTPS address")
-	discoveryCmd.PersistentFlags().StringVar(&serverArgs.DiscoveryOptions.GrpcAddr, "grpcAddr", ":15010",
-		"Discovery service grpc address")
+	discoveryCmd.PersistentFlags().StringVar(&serverArgs.DiscoveryOptions.GrpcAddr, "gRPCAddr", ":15010",
```

Comment:
you cannot change this, its part of the API and needs to be backwards compatible